doc/classes/int.xml

```diff
@@ -198,169 +195,168 @@
 			<description>
 				Raises an [int] to a power of a [float]. The result is a [float].
 				[codeblock]
-				print(8**0.25) # 1.68179283050743
+				print(2**0.5) # Prints 1.4142135623731
 				[/codeblock]
 			</description>
 		</operator>
 		<operator name="operator **">
 			<return type="int" />
 			<param index="0" name="right" type="int" />
 			<description>
-				Raises an [int] to a power of a [int].
+				Raises the left [int] to a power of the right [int].
 				[codeblock]
-				print(5**5) # 3125
+				print(3**4) # Prints 81
 				[/codeblock]
 			</description>
 		</operator>
 		<operator name="operator +">
 			<return type="float" />
 			<param index="0" name="right" type="float" />
 			<description>
-				Adds an [int] and a [float]. The result is a [float].
+				Adds the [int] and the [float]. The result is a [float].
 			</description>
 		</operator>
 		<operator name="operator +">
 			<return type="int" />
 			<param index="0" name="right" type="int" />
 			<description>
-				Adds two integers.
+				Adds the two [int]s.
 			</description>
 		</operator>
 		<operator name="operator -">
 			<return type="float" />
 			<param index="0" name="right" type="float" />
 			<description>
-				Subtracts a [float] from an [int]. The result is a [float].
+				Subtracts the [int] by the [float]. The result is a [float].
```

Comment:
This is not a correct usage of the verb "subtract". The previous version was correct in using "subtract a float from".

Reply:
Oops, didn't know this form doesn't exist. It's annoying you can't say this left-to-right in any way.